e2e-app/src/app/timepicker/navigation/timepicker-navigation.po.ts

```diff
@@ -0,0 +1,11 @@
+import {$} from 'protractor';
+
+export type Field = 'hour' | 'minute' | 'second';
+
+export class Page {
```

Comment:
Could you please rename to `TimepickerPage`, please?
- all other pages have specific names
- having only `Page` makes navigation by class name difficult